src/BlazorWebView/tests/MauiDeviceTests/WebViewHelpers.Windows.cs

```diff
@@ -12,6 +12,17 @@ public static async Task WaitForWebViewReady(WebView2 wv2)
 		{
 			CoreWebView2 coreWebView2 = null;
 
+			// Ensure that the WebView2 runtime is installed and initialized and has a CoreWebView2 instance.
```

Comment:
@HarishKumarSF4517 There are not related changes, probably related with rebase. Could you take a look?